package.json

```diff
@@ -3,7 +3,7 @@
   "version": "9.3.0",
   "description": "NHS.UK frontend contains the code you need to start building user interfaces for NHS websites and services.",
   "engines": {
-    "node": ">=20.0.0"
+    "node": "^22.11.0"
```

Comment:
Should we continue to maintain compatibility with Node 20, or are there any features of Node 22 that would mean we’d want everyone to upgrade?

(This will affect the prototype kit too, as well as production services)